driver/normalizer/normalizer.go

```diff
@@ -90,26 +103,34 @@ var Normalizers = []Mapping{
 
 	MapSemantic("CPPASTImplicitName", uast.Identifier{}, MapObj(
 		Obj{
-			"Name": Var("name"),
+			"Name":                 Var("name"),
+			"IsQualified":          Var("ignIsQual"),
+			"IsAlternate":          Var("ignIsAlternate"),
+			"IsOverloadedOperator": Var("ignIsOver"),
 		},
 		Obj{
 			"Name": Var("name"),
 		},
 	)),
 
-	MapSemantic("CPPASTQualifiedNames", uast.QualifiedIdentifier{}, MapObj(
-		Obj{
-			"Prop_AllSegments": Var("names"),
-		},
-		Obj{
-			"Names": Var("names"),
-		},
-	)),
+	// Disabled: parts can be anything, not only identifiers (like function calls or object instantiations)
+	// and the SDK always expects identifiers (but should except Any[] for this object)
+	//MapSemantic("CPPASTQualifiedName", uast.QualifiedIdentifier{}, MapObj(
+	//	Obj{
+	//		"Prop_AllSegments": Var("names"),
+	//		"IsQualified": Var("ignIsQual"),
+	//		"IsConversionOperator": Var("ignIsConv"),
+	//		"IsFullyQualified": Var("ignFullyQual"),
+	//	},
+	//	Obj{
+	//		"Names": Var("names"),
+	//	},
+	//)),
 
 	MapSemantic("CPPASTFunctionDefinition", uast.FunctionGroup{}, MapObj(
 		Fields{
-			//{Name: "IsDefaulted", Optional: "optDefaulted", Op: Var("ignDefaulted")},
-			//{Name: "IsDeleted", Optional: "optDeleted", Op: Var("ignDeleted")},
+			{Name: "IsDefaulted", Optional: "optDefaulted", Op: Var("ignDefaulted")},
```

Comment:
As you know, I'm not a fan of dropping fields :)

There are only three valid options here:
1) You expect the field to always be `nil`/`false`, so you can use `Is(nil)` or `Bool(false)`.
2) You know that the field value varies, but don't know what to do with it. Unfortunately we don't have every feature defined in Semantic UAST, so for now I think it's okay to make a fake node in `FunctionGroup` and dump those values to it. But please **don't drop fields** like this.
3) The node is a complete garbage and just duplicates the info in the main node (happens with JS and Python). Then it's the only one case when you can drop it with `Any(nil)` operation.

Reply:
I personally don't like any of these options:

1. They aren't always nil/false, they can be true or false.
2. This is the case with these fields. I don't think dumping them into a fake node in the `FunctionGroup` will be of much help (or pretty). Also, in this AST they're not of a single node, but of almost all nodes that form the `FunctionGroup`: each argument, return values, etc. In this case, we would need many fake nodes, which is even uglier.
3. Not the case.

In my opinion, dropping fields that doesn't have a representation in the defined schema of the semantic object is not bad. The user always can go back to the non semantic UAST or even the native one to get those. 

But ideally every node in the semantic UAST would have a child node of type "SemNodeExtension" or something like that storing all the properties that doesn't conform to the parent node schema. This way we don't lost anything but we don't pollute the main group with fake nodes at random locations.

Reply:
As discussed on the other PR, I changed those fields to use `AnyNode(nil)` to drop them until we decide on a better solution in the SDK for these unmatched properties.